envoy/tracing/trace_driver.h

```diff
@@ -5,6 +5,7 @@
 #include <vector>
 
 #include "envoy/common/pure.h"
+#include "envoy/data/accesslog/v3/accesslog.pb.h"
```

Comment:
so this introduces the dependency on access_logs *in* tracing/trace_driver? I understand how this ends up here but seems a bit weird to me.

Reply:
I also considered this dependency seems weird, then submitted refactor PR. https://github.com/envoyproxy/envoy/pull/18942#issuecomment-965378956
But, maintainers referred to me utilizing the current API.